app/controllers/home_controller.rb

```diff
@@ -1,6 +1,28 @@
+require 'rss'
+require 'open-uri'
+
 class HomeController < ActionController::Base
   layout 'application'
 
   def index
+    @status = Rails.cache.read("rave_status")
   end
+
+  def send_alert
+    if request[:status] == "on"
+      status = "off"
+    else
+      status = "on"
+    end
+
+    Rails.cache.write("rave_status", status)
+    redirect_to :action => 'index'
+  end
+
+  def live_feed
```

Comment:
wasn't sure if we wanted to stream the actual feed anywhere so I left it, but I'll probably remove it since I didn't end up putting it anywhere